gravitee-apim-console-webui/src/portal/customization/api/api-list/portal-api-list.component.html

```diff
@@ -0,0 +1,149 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (form) {
+  <form [formGroup]="form">
+    <div class="menu-api-container">
+      <mat-card class="menu-api">
+        <mat-card-content class="categories__content">
+          <div class="menu-api__content__header">
+            <div class="menu-api__content__header__title">
+              <h3>API Subscription</h3>
+              <span class="gio-badge-warning"> <mat-icon svgIcon="gio:chat-lines"></mat-icon>&nbsp;Applies to both portals </span>
+            </div>
+            <div class="menu-api__content__header__description">Customize the information included in API header</div>
+          </div>
+          <div>
+            <mat-form-field appearance="outline" class="mat-formfield">
+              <input matInput formControlName="apiKeyHeader" />
+              <mat-label>API Key Header</mat-label>
+            </mat-form-field>
+          </div>
+          <div>
+            <mat-hint
+              >Used to display CURL commands in the Developer Portal. To insure valid CURL commands, make sure this value matches the
+              defined API Key Header in the Gateway YAML configuration</mat-hint
+            >
+          </div>
+        </mat-card-content>
+      </mat-card>
+      <mat-card class="menu-api">
+        <mat-card-content>
+          <div class="menu-api__content__header">
+            <div class="menu-api__content__header__title-row">
+              <div>
+                <div class="menu-api__content__header__title">
+                  <h3>API Details</h3>
+                  <span class="gio-badge-warning"> <mat-icon svgIcon="gio:chat-lines"></mat-icon>&nbsp;Applies to both portals </span>
+                </div>
+                <div class="menu-api__content__header__description">Customize the information displayed on the API details page</div>
+              </div>
+              <button
+                class="add-button"
+                (click)="createHeader()"
+                mat-raised-button
+                color="primary"
+                *gioPermission="{ anyOf: ['environment-api-u'] }"
```

Comment:
I think this would probably be `environment-settings-u` --> You can double check the resource in the backend to verify.

Reply:
Done